fuel-core/src/schema/node_info.rs

```diff
@@ -0,0 +1,76 @@
+use super::scalars::U64;
+use crate::service::Config;
+use async_graphql::{Context, Object};
+
+#[derive(Default)]
+pub struct NodeQuery {}
+
+pub struct NodeInfo {
+    utxo_validation: bool,
+    predicates: bool,
+    vm_backtrace: bool,
+    min_gas_price: U64,
+    min_byte_price: U64,
+    max_tx: U64,
+    max_depth: U64,
+}
+
+#[Object]
+impl NodeInfo {
+    async fn utxo_validation(&self) -> bool {
+        self.utxo_validation
+    }
+
+    async fn predicates(&self) -> bool {
+        self.predicates
+    }
+
+    async fn vm_backtrace(&self) -> bool {
+        self.vm_backtrace
+    }
+
+    async fn min_gas_price(&self) -> U64 {
+        self.min_gas_price
+    }
+
+    async fn min_byte_price(&self) -> U64 {
+        self.min_byte_price
+    }
+
+    async fn max_tx(&self) -> U64 {
+        self.max_tx
+    }
+
+    async fn max_depth(&self) -> U64 {
+        self.max_depth
+    }
+}
+
+#[Object]
+impl NodeQuery {
+    async fn node_version(&self, _ctx: &Context<'_>) -> async_graphql::Result<String> {
+        const VERSION: &str = env!("CARGO_PKG_VERSION");
+
+        Ok(VERSION.to_owned())
+    }
```

Comment:
We already have this as `Query.version`: https://github.com/FuelLabs/fuel-core/blob/master/fuel-core/src/schema/tx.rs#L31-L35

I think we should remove that and keep the new `node_version`.

Reply:
yeah I actually copy/pasted from it, forgot to remove it